src/lib/utils/events.ts

```diff
@@ -433,6 +436,12 @@ export class EventDispatcher {
         context?: any,
         priority?: number
     ): this;
+    on(
```

Comment:
I'd rather not do this for now - better to have none of the events typed than some of them. Eventually it'd be nice to have autocomplete, but that'll probably not come until removing `@Component`...